src/Silverback.Integration/Messaging/Configuration/IBrokerEventsHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using System;
+using System.Threading.Tasks;
+
+namespace Silverback.Messaging.Configuration
+{
+    /// <summary>
+    ///     Exposes some event hooks that are being called during configuration and initialization
+    ///     of brokers.
+    /// </summary>
+    public interface IBrokerEventsHandler
+    {
+        /// <summary>
+        ///     Is triggered when all endpoints were configured.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        Task EndpointsConfiguredAsync();
```

Comment:
This should instead be called `OnEndpointsConfiguredAsync`? I don't know. I would personally prefix the event handlers with `On` but I don't know if that's a common thing.

Reply:
We agreed to add the `On` prefix.